src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -54,14 +58,20 @@ public void ldpcvOfLdprsMustSupportPostWithoutMementoDatetimeHeader(final String
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-post",
                                         ps);
 
+        // create the versioned resource
+        final String newResource = createVersionedResource(uri, info).body().asString();
+        // create a memento for the current time
+        final Response versionResponse = doPost(newResource + "/fcr:versions");
```

Comment:
I do not believe we can expect that other Fedora implementations will return memento headers in the response to the creation of a new memento.
You should probably make a subsequent `HEAD` or `GET` request in order to check for the memento header.

Reply:
Ah, right!  I'm going to wait for Danny's PR to go in and then use his createMemento function here.